ge04_team01/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    is_new_customer = fields.Boolean(related="partner_id.is_new_customer")
+
+    def apply_discount(self):
+        for sale_order in self:
```

Comment:
`for order in self` is widely used on standard. It is not incorrect what you did but the standard way is the preferred way.